stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewGroupDescriptor.java

```diff
@@ -16,11 +16,19 @@
 import com.facebook.stetho.common.android.FragmentCompatUtil;
 import com.facebook.stetho.inspector.elements.AbstractChainedDescriptor;
 
+import java.lang.ref.WeakReference;
 import java.util.Collections;
 import java.util.Map;
 import java.util.WeakHashMap;
 
 final class ViewGroupDescriptor extends AbstractChainedDescriptor<ViewGroup> {
+
+  /**
+   * This is a marker object for values in the {@link #mViewToElementMap}. When this is present as a
+   * value the key should be used as the value.
+   */
+  private static final Object SENTINEL = new Object();
```

Comment:
let's make this non-static. that way it can be GC'd once Chrome disconnects. (descriptors are effectively singletons anyway)

```
private final Object mSentinal = new Object();
```

Heck we could even save the allocation and initialize mSentinel to equal `this` in the constructor :) ...